Classes/Emogrifier.php

```diff
@@ -323,6 +334,81 @@ protected function process(\DOMDocument $xmlDocument)
     }
 
     /**
+     * Applies $newStyles to $node.
+     *
+     * This method maps CSS styles to HTML attributes and adds those to the
+     * node.
+     *
+     * @param string[] $newStyles the new CSS styles taken from the global styles
+     *                            to be applied to this node.
+     * @param \DOMNode $node      node to apply styles to.
+     *
+     * @return void
+     */
+    private function mapStyles2HtmlAttrs($newStyles, \DOMNode $node)
```

Comment:
Please add the "array" type hinting for the first parameter (as part of the declaration).